app/src/main/java/com/sun/ev_dictionary/ui/common/CommonActivity.kt

```diff
@@ -22,15 +28,34 @@ class CommonActivity : BaseActivity<ActivityCommonSearchBinding>() {
     private fun replaceDetailFragment(fragmentName: String) {
         when (fragmentName) {
             VESearchFragment::class.java.simpleName -> {
-                ActivityUtils.replaceFragment(
-                    supportFragmentManager,
-                    R.id.frameContainer,
-                    VESearchFragment.newInstance()
-                )
+                replaceFragment(VESearchFragment.newInstance())
+            }
+            EssentialFragment::class.java.simpleName -> {
+                replaceFragment(EssentialFragment.newInstance())
+            }
+            IrregularFragment::class.java.simpleName -> {
+                replaceFragment(IrregularFragment.newInstance())
+            }
+            IeltsFragment::class.java.simpleName -> {
+                replaceFragment(IeltsFragment.newInstance())
+            }
+            ToeicFragment::class.java.simpleName -> {
+                replaceFragment(ToeicFragment.newInstance())
+            }
+            ToeflFragment::class.java.simpleName -> {
+                replaceFragment(ToeflFragment.newInstance())
             }
         }
     }
 
+    private fun replaceFragment(fragment: Fragment) {
+        ActivityUtils.replaceFragment(
```

Comment:
thay vì viết utils e có thể viết extention cho AppCompatActivity như thế này
và lần tới ở activity nào extend từ AppCompatActivity cũng có thể sử dụng

```suggestion
        fun AppCompatActivity.replaceFragment(resourceId: Int, fragment: Fragment) {
    supportFragmentManager.beginTransaction().replace(resourceId, fragment)
}
```